sdk/keyvault/keyvault-keys/src/keyVaultCryptographyClient.ts

```diff
@@ -171,28 +172,22 @@ export class KeyVaultCryptographyClient {
       span.end();
     }
 
-    return { result: result.result!, algorithm, keyID: this.getKeyID() };
+    return {
+      algorithm: encryptParameters.algorithm,
+      result: result.result!,
+      keyID: this.getKeyID(),
+      additionalAuthenticatedData: result.additionalAuthenticatedData,
+      authenticationTag: result.authenticationTag,
+      iv: result.iv
```

Comment:
We're still not throwing when the `iv` is unnecessary! Let's catch up, I want to understand.

Reply:
No problem! 😄 As Heath mentioned we really don't want to validate service parameters to avoid tightly coupling to our service. This also lines up well with the [typescript guidelines](https://azure.github.io/azure-sdk/typescript_implementation.html#general-parameter-validation). The error that comes back from the service is descriptive and can change without having to update all the clients so I think it's the right thing to do